app/http.php

```diff
@@ -1057,9 +1057,10 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
 
             // Error occured
             if ($executionResponse['errNo'] !== 0) {
-                // Unknown protocol error code, but also means parsing issue
+                // 7102 is unknown protocol error code, but also means parsing issue.
+                // 27 is out of memory error code.
                 // As patch, we consider this too big entry for headers (logs&errors)
-                if ($executionResponse['errNo'] === 7102) {
+                if ($executionResponse['errNo'] === 7102 || $executionResponse['errNo'] === 27) {
```

Comment:
I think you should have a separate condition and relevant error message for when the exit code is 27 🤔 
Are we absolutely sure that 27 can occur only because of large logs? If not, the exception message is a bit misleading

Reply:
Large logs ar eno longer a problem, and should never happen. Making myself task to remove large log exception before release.